main.tscn

```diff
@@ -1,13 +1,12 @@
-[gd_scene load_steps=14 format=3 uid="uid://dfhenhhamjbxb"]
```

Comment:
changes to main can be ignored, they are some rearrangement for testing // can be overridden if there were conflicts with other branches